### PR TITLE
ci: uncomment audit

### PR DESCRIPTION
**Pull request by hiroppy**
- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [ ] This is a **code refactor**
- [ ] This is a **test update**
- [ ] This is a **docs update**
- [x] This is a **metadata update**

### For Bugs and Features; did you add new tests?

no

### Motivation / Use-Case

I hope the audit has been fixed on v4.

### Breaking Changes

no

### CODE REVIEW

**Comment by hiroppy** [comment]
hmm, need to update yargs-parser..

**Comment by hiroppy** [comment]
@evilebottnawi please review this

File: azure-pipelines.yml [hunk 1]
```diff
@@ -32,8 +32,8 @@ jobs:
         displayName: 'Install dependencies'
       - script: npm run lint
         displayName: 'Run lint'
-      # - script: npm run security
-      #   displayName: 'Run NPM audit'
+      - script: npm audit --production
+        displayName: 'Run NPM audit'
```
Comment (evilebottnawi) [inline azure-pipelines.yml:36]: Why only production? In theory dev dependencies can be vulnerable too, but at the moment we’re not auditing anything at all, so let's merge

Comment (hiroppy) [reply azure-pipelines.yml:36]: right, but currently we can't fix yargs-parser issue. https://github.com/webpack/webpack-dev-server/pull/2636#issuecomment-638170801

**Pull request merged**